lib/testing/gpuTrackingBeamformSimulate.hpp

```diff
@@ -105,14 +105,14 @@ class gpuBeamformPulsarSimulate : public kotekan::Stage {
     bool update_phase;
 
     /// Brute-force beamform by multiplying each element with a phase shift
-    void cpu_beamform_pulsar(double* input, double* phase, float* output, int nsamp, int nelem,
-                             int npsr, int npol);
+    void cpu_beamform_tracking(double* input, double* phase, float* output, int nsamp, int nelem,
```

Comment:
```suggestion
    void cpu_tracking_beamformer(double* input, double* phase, float* output, int nsamp, int nelem,
```
In other places we used "tracking beamformer", we should be consistent with the order of names if possible.